File: src/fib/java/fib.java
public class fib {
    static int fib( int x ) {
	if( x < 2 ) {
	    return 1;
	} else {
	    return fib( x - 1 ) + fib( x - 2 );
	}
    }

    public static void main( String[] args ) {
	int y = 0;

	for( int i = 0; i < 300; i++ ) {
	    y += fib( 30 );
	}

	System.out.println( fib( 30 ) + "  / " + (y/10) );
    }
}

File: src/fib/c/fib.c
#include <stdio.h>
#include <stdlib.h>

static long res = 0;

static long fib(long x) {
   if (x < 2) {
      return 1;
   } else {
      return fib(x - 1) + fib(x - 2);
   }
}

static void doit(int num) {
   while (num > 0) {
      res = fib(35);
      num--;
   }
}

int main(int argc, char *argv[]) {
   doit(900);
   fprintf(stderr, "%ld\n", res);
   return res != 14930352;
}
